.github/workflows/ci.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Set up Go 1.x
         uses: actions/setup-go@v4.0.0
         with:
-          go-version: '1.20'
+          go-version: '1.21'
```

Comment:
Any reason we need to update the go version in this PR?

Reply:
I have mentioned this in slack message -https://intuit.enterprise.slack.com/archives/C05K82P3N9J/p1708410520173079